### PR TITLE
Added GSSAPI auth support

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@jakirkham Any idea what is wrong with the py36 windows builds?  Do we even need to make them since this is ultimately about building postgresql

**Comment by mariusvniekerk** [comment]
@ericdill can you merge this?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,12 @@ source:
...
+    - timezone.patch  # [win and (py >= 35)]
     # https://www.postgresql.org/message-id/attachment/43919/0002-Support-parallel-build-with-MSBuild.patch
     - symbol_tempfile.patch  # [win]
 
 build:
-  number: 2
+  number: 3
   features:
     - vc9  # [win and py27]
     - vc10  # [win and py34]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Needs a `vc14` feature for `py36` as well.

Note: Sorry for the weird comment location. GitHub wouldn't let me put it a line lower.

Comment (mariusvniekerk) [inline recipe/meta.yaml]: ah

Comment (mariusvniekerk) [inline recipe/meta.yaml]: should i can remove the py34 one?

Comment (jakirkham) [inline recipe/meta.yaml]: Sure, go for it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,41 +11,45 @@ source:
...
+    - vc14  # [win and py >= 35]
 
 requirements:
   build:
     - m2-bison         # [win]
     - m2-diffutils     # [win]
     - m2-flex          # [win]
     - openssl 1.0.*
-    - perl   >=5.20    # [win]
+    - perl   >=5.22    # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: FWIW we have just been pinning this to a specific `perl` version elsewhere. So maybe set this to `5.22.2.1`. This way it won't add unneeded additional `perl`s after re-rendering.

**Comment by mariusvniekerk** [comment]
@jakirkham looks okay now?

**Comment by mariusvniekerk** [comment]
@jakirkham can you merge?  I don't have permissions here

**Comment by mariusvniekerk** [comment]
@msarahan ping

**Comment by mariusvniekerk** [comment]
@ocefpaf ping?

**Pull request merged**